CONTRIBUTING.md

```diff
@@ -43,8 +43,10 @@ $ git remote add upstream git://github.com/nodejs/node.git
 
 #### Which branch?
 
-For developing new features and bug fixes, the `master` branch should be pulled
-and built upon.
+For developing new features and bug fixes, the `master` branch should be
+pulled and built upon.  See the section on
```

Comment:
The use of "should" in this first sentence has always bothered me. The master branch *is* the branch for new features and bug fixes, not *should* be. So perhaps changing this to:

```
New features and bug fixes are landed in `master` first, then backported to release
branches. See the section on [backporting]...
```